Teams.md

```diff
@@ -10,32 +10,28 @@ The Org Maintainers are a team defined in the [governance](governance/governance
 
-This team can be directly contacted at the private address of cncf-helm-maintainers@lists.cncf.io. The members of this team are documented in the MAINTAINERS.md file in this repository.
+This team can be directly contacted at the private address of <cncf-helm-maintainers@lists.cncf.io>. The members of this team are documented in the [MAINTAINERS.md](MAINTAINERS.md) file in this repository.
 
 ## Security Team
 
-Helm has a security team and process for handling all security issues. This team, the security process, and the method to contact them (including with the use of encryption) is documented in the SECURITY.md file.
+Helm has a security team and process for handling all security issues. This team, the security process, and the method to contact them (including with the use of encryption) is documented in the [SECURITY.md](SECURITY.md) file in this repository.
 
-## Helm Client
+## Helm Client and SDK
 
-The [Helm client/CLI](https://github.com/helm/helm) has a dedicated team maintaining it. The members of that team are documented in the [OWNERS](https://github.com/helm/helm/blob/main/OWNERS) file in the root of the project.
+The [Helm client/CLI and SDK](https://github.com/helm/helm) has a dedicated team maintaining it. The members of that team are documented in the [OWNERS](https://github.com/helm/helm/blob/main/OWNERS) file in the root of the project.
 
-This team can be directly contacted at the private address of cncf-helm-core-maintainers@lists.cncf.io.
+This team can be directly contacted at the private address of <cncf-helm-core-maintainers@lists.cncf.io>.
 
 ## Charts Maintainers
 
-The Charts Maintainers handle numerous repositories in the Helm GitHub organization. Specifically, those that deal with the [charts](https://github.com/helm/charts) repository, testing charts, the [hub](https://github.com/helm/hub), etc.
+The Charts Maintainers handle numerous repositories in the Helm GitHub organization. Specifically, those that deal with Helm charts such as chart [testing](https://github.com/helm/chart-testing), [releasing](https://github.com/helm/chart-releaser), [automation](https://github.com/orgs/helm/repositories?q=action), and [documentation](https://github.com/helm/charts-repo-actions-demo).
 
-The charts maintainers are documented in the [OWNERS](https://github.com/helm/charts/blob/master/OWNERS) file in the root of the charts repository. They can be contacted privately at kubernetes-charts-leads@googlegroups.com.
-
-## Monocular
-
-[Monocular](https://github.com/helm/monocular) is a web application to present charts that are in chart repositories. It us used to power the Helm Hub. The maintainers are listed in the [OWNERS](https://github.com/helm/monocular/blob/master/OWNERS) file in the root of the project.
+The charts maintainers are documented in the [OWNERS](https://github.com/helm/charts/blob/master/OWNERS) file in the root of the charts repository.
```

Comment:
https://github.com/helm/charts is archived. Should be removed from here I think?

(Can either remove in this PR, or a followup PR to remove)